CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
 # Change Log
 
+## [0.9.0](https://github.com/confluentinc/ksql/releases/tag/v0.9.0-ksqldb) (2020-05-07)
+
+### Features
```

Comment:
Would be worth separating out the truly user visible features like multi-way joins and the improvements on the `LIKE` clause.

Reply:
Done. 

I put them at the top of the list.